Documentation/dev/developer-guide.md

```diff
@@ -136,6 +136,11 @@ When committing new dependencies, please use the following guidelines:
   - `Gopkg.toml: Add new dependency $your_new_dependency`
   - `Gopkg.lock,vendor: Add new dependency $your_new_dependency`
 
+## Helm templates
+
+If you have added a new Helm Chart and would like to render the template to check the values and nesting within the yaml file you can run:
+``` helm template CHART_DIR -x PATH_TO_TEMPLATE/file.yaml ```
```

Comment:
The command should go onto its own line, with backticks on their own line as well. Look at the other examples of commands in the documentation for reference.